scripts/install_openssl.sh

```diff
@@ -36,7 +36,6 @@ pushd ${SRCDIR}
 # Build the library.
 ${ARCH_PROG} ./config --prefix=${INSTALLDIR} \
                       --debug \
-                      enable-fuzz-libfuzzer \
```

Comment:
The ossfuzz code seems to indicate that `--with-fuzzer-lib=/usr/lib/libFuzzingEngine` might be necessary instead of removing this. See https://github.com/google/oss-fuzz/blob/master/projects/openssl/build.sh#L18